src/containers/ProfileDetails/components/CaptivePortal/components/Users/index.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { Card, Table } from 'antd';
+import Button from 'components/Button';
+import Modal from 'components/Modal';
+import { FormOutlined, DeleteOutlined } from '@ant-design/icons';
+import FormModal from '../FormModal';
+import styles from '../../../index.module.scss';
+
+const Users = ({ userList, handleAddUser, handleUpdateUser, handleDeleteUser }) => {
+  const [addModal, setAddModal] = useState(false);
+  const [editModal, setEditModal] = useState(false);
+  const [deleteModal, setDeleteModal] = useState(false);
+  const [activeUser, setActiveUser] = useState({});
+  const [tableData, setTableData] = useState(userList);
```

Comment:
this should be controlled in parent, too many different states/props of same data